MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -119,7 +119,11 @@ class RouteManeuverViewController: UIViewController {
     
     public func updateStreetNameForStep() {
         if let name = step?.names?.first {
-            streetLabel.unabridgedText = name
+            if let step = step, let ref = step.codes?.first, ["i", "us"].contains(ref.lowercased().components(separatedBy: " ").first!) {
+                streetLabel.unabridgedText = ref.components(separatedBy: " ").joined(separator: "-")
```

Comment:
U.S. Routes are written “US 123” in most states. Only a few states like Oklahoma hyphenate them.